cmd/clean.go

```diff
@@ -26,15 +26,25 @@ to quickly create a Cobra application.`,
 
 		log.Println("removing $HOME/.kubefirst and $HOME/.flare")
 		// todo ask for user input to verify?
```

Comment:
Maybe should we use the configs? instead of the name in the log string...

Reply:
that's true, It was updated